src/tests/ftest/server/metadata.yaml

```diff
@@ -47,7 +48,16 @@ pool:
   svcn: 5
   scm_size: 1G
   control_method: dmg
-  properties: svc_ops_enabled:1,svc_ops_entry_age:150
+  pool_prop: !mux
+    svc_ops_60:
+      properties: svc_ops_enabled:1,svc_ops_entry_age:60
+# Uncomment following for manual test
```

Comment:
Also might add that the values of created_cont_min and created_cont_max will be affected by the duration of the svc_ops_entry_age, so they may need to be adjusted accordingly.